public_html/admin/controller/responses/design/blocks_manager.php

```diff
@@ -151,16 +151,15 @@ public function block_info()
         //expect only 1 block details per layout
         $this->data = array_merge($info[0], $this->data);
         $this->data['block_info'] = $info;
-
-        //get specific description 
+        //get specific description
         if ($custom_block_id > 0) {
             $descr = $lm->getBlockDescriptions((int)$custom_block_id);
             $language_id = $this->language->getContentLanguageID();
             $this->data['title'] = $descr[$language_id]['title'];
             $this->data['description'] = $descr[$language_id]['description'];
 
             //detect edit URL and build button
-            if ($this->data['block_txt_id'] == 'html_block' || $this->data['block_txt_id'] == 'listing_block') {
+            if ($this->data['block_txt_id'] == 'html_block' || $this->data['block_txt_id'] == 'listing_block' || $this->data['block_txt_id'] == 'custom_form_block') {
```

Comment:
Array with block_txt_ids must be part $this->data to add ability to extend it.
Add this into constructor of this controller
$this->data['allowed_txt_ids] = [''html_block'',  'block_txt_id', 'custom_form_block'];

and then rewrite condition with in_array()